### PR TITLE
[stable9] changed ownCloud to Nextcloud, updated config options

### PR DESCRIPTION
**Pull request by palenhart**
https://github.com/nextcloud/documentation/commit/e917b2ab52c698a1591e9158706381bc94aef0f1 seems to be generated from this file and reverted all occurrences of Nextcloud back to ownCloud.
I replaced this config sample with the config sample from the master branch.

It would be great if someone could check whether all changes to available parameters from master are also true for stable9 and the changes in my commit are correct :smile: @MorrisJobke @LukasReschke

### CODE REVIEW

File: config/config.sample.php [hunk 1]
```diff
@@ -1233,14 +1248,32 @@
...
+ *
+ * This is a property used by the clients to find out if a backup has been
+ * restored on the server. Once a backup is restored run
+ * ./occ maintenance:data-fingerprint
+ * To set this to a new value.
+ *
+ * Updating/Deleting this value can make connected clients stall until
+ * the user has resolved conflicts.
+ */
+'data-fingerprint' => '',
```
Comment (LukasReschke) [inline config/config.sample.php]: Not relevant for stbale9

**Comment by LukasReschke** [comment]
LGTM besides my small remark 🚀

File: config/config.sample.php [hunk 2]
```diff
@@ -195,11 +194,18 @@
 'session_keepalive' => true,
 
 /**
+ * Enforce token authentication for clients, which blocks requests using the user
+ * password for enhanced security. Users need to generate tokens in personal settings
+ * which can be used as passwords on their clients.
+ */
+'token_auth_enforced' => false,
```
Comment (MorrisJobke) [inline config/config.sample.php]: This is not available in stable9. Could you remove it?

File: config/config.sample.php [hunk 3]
```diff
@@ -962,6 +935,30 @@
...
+
+	// Turn on consistent hashing
+	\Memcached::OPT_LIBKETAMA_COMPATIBLE => true,
+
+	// Enable Binary Protocol
+	\Memcached::OPT_BINARY_PROTOCOL =>      true,
+
+	// Binary serializer vill be enabled if the igbinary PECL module is available
+	//\Memcached::OPT_SERIALIZER => \Memcached::SERIALIZER_IGBINARY,
+),
```
Comment (MorrisJobke) [inline config/config.sample.php]: Was added in master -> please remove this from stable9

File: config/config.sample.php [hunk 4]
```diff
@@ -972,14 +969,22 @@
...
 
 /**
- * Using Object Store with ownCloud
+ * TTL of chunks located in the cache folder before they're removed by
+ * garbage collection (in seconds). Increase this value if users have
+ * issues uploading very large files via the Nextcloud Client as upload isn't
+ * completed within one day.
+ */
+'cache_chunk_gc_ttl' => 86400, // 60*60*24 = 1 day
+
```
Comment (MorrisJobke) [inline config/config.sample.php]: Was added in master -> please remove this from stable9

File: config/config.sample.php [hunk 5]
```diff
@@ -1225,6 +1231,15 @@
 'filelocking.enabled' => true,
 
 /**
+ * Set the time-to-live for locks in secconds.
+ *
+ * Any lock older than this will be automatically cleaned up.
+ *
+ * If not set this defaults to either 1 hour or the php max_execution_time, whichever is higher.
+ */
+'filelocking.ttl' => 3600,
```
Comment (MorrisJobke) [inline config/config.sample.php]: Was added in master -> please remove this from stable9

File: config/config.sample.php [hunk 6]
```diff
@@ -1233,7 +1248,12 @@
 'memcache.locking' => '\\OC\\Memcache\\Redis',
 
 /**
- * Set this ownCloud instance to debugging mode
+ * Disable the web based updater
+ */
+'upgrade.disable-web' => false,
```
Comment (MorrisJobke) [inline config/config.sample.php]: Was added in master -> please remove this from stable9

**Comment by MorrisJobke** [comment]
@Zollak Thanks a lot :) Once my remarks are fixed this can get merged :)

**Comment by Zollak** [comment]
Thank you for reviewing :smile: 
I removed all the non-available options.

**Comment by MorrisJobke** [comment]
Nice 👍

**Pull request merged**